extract.rb

```diff
@@ -1,6 +1,19 @@
 require 'open-uri'
 
-file = open(ARGV[0]).read
+target = ARGV[0]
+
+if target.nil? or target.empty?
+   puts "Looks like you forgot to supply file path or Url to the target. Kindly specify the target "
+   exit
+end
+
+puts "Processing target #{target} .............. "
```

Comment:
Let's make this optional, based on `ARGV[1] == '-c'`, so that it's easy to pipe the raw output to a separate file or command that takes the paths as input.

Reply:
Okay. 
If c argument is passed then only verbose output will be enabled. 
How about using `v` (as in verbose) for the flag.